### DISCUSSION TITLE
Github Latex not rendering properly

### DISCUSSION THREAD

**Discussion by lokehagberg** [community/community#31163] [Other Feature Feedback, Questions, & Ideas]
Example: $works$

$f + a + i = l + s$

$$w*o*r*k*s$$

This is a variable $fails$, why does it not render $p$?

Neither does this o $n$  e!

At least it does not render in the Preview.

**Comment by F4NT0** [comment]
Can you show the code you are using to generate this letters?

**Reply by lokehagberg** [reply]
![GithubLatexProblem](https://user-images.githubusercontent.com/48260753/188160302-d8ea70e9-b66e-49b3-b581-e76eff5ffbfc.PNG)

**Reply by lokehagberg** [reply]
The preview I had when I created the thread has changed, so things that did not work then does work now.
